platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/ShadowGraphTest.java

```diff
@@ -134,40 +133,40 @@ private void initShadowgraph(final Random random, final int numEvents, final int
     @ParameterizedTest
     @MethodSource("graphSizes")
     void testFindAncestorsForMultipleEvents(final int numEvents, final int numNodes) {
-        Random random = RandomUtils.getRandomPrintSeed();
+        final Random random = RandomUtils.getRandomPrintSeed();
 
         initShadowgraph(random, numEvents, numNodes);
 
-        Set<ShadowEvent> generatedShadows =
+        final Set<ShadowEvent> generatedShadows =
                 generatedEvents.stream().map(shadowgraph::shadow).collect(Collectors.toSet());
 
-        Set<ShadowEvent> generatedShadowsSubset = generatedShadows.stream()
+        final Set<ShadowEvent> generatedShadowsSubset = generatedShadows.stream()
                 .filter((hash) -> random.nextDouble() < 0.5)
                 .collect(Collectors.toSet());
 
-        Set<Hash> actualAncestors = shadowgraph.findAncestors(generatedShadowsSubset, (e) -> true).stream()
+        final Set<Hash> actualAncestors = shadowgraph.findAncestors(generatedShadowsSubset, (e) -> true).stream()
                 .map(ShadowEvent::getEventBaseHash)
                 .collect(Collectors.toSet());
 
-        for (ShadowEvent shadowEvent : generatedShadowsSubset) {
+        for (final ShadowEvent shadowEvent : generatedShadowsSubset) {
             assertSetsContainSameHashes(ancestorsMap.get(shadowEvent.getEventBaseHash()), actualAncestors);
         }
     }
 
     @RepeatedTest(10)
     void testFindAncestorsExcludesExpiredEvents() {
-        Random random = RandomUtils.getRandomPrintSeed();
+        final Random random = RandomUtils.getRandomPrintSeed();
         initShadowgraph(random, 100, 4);
 
-        long expireBelowGen = random.nextInt(10) + 1;
+        final long expireBelowGen = random.nextInt(10) + 1;
 
         final NonAncientEventWindow eventWindow = new NonAncientEventWindow(
                 0 /* ignored by shadowgraph */, 0 /* ignored by shadowgraph */, expireBelowGen, GENERATION_THRESHOLD);
```

Comment:
It looks like this unit test only checks generation based threshold.  The issue for making the the shadowgraph compatible with birthRound was closed as completed, but these unit tests were not converted over to testing.  

Created a new issue for the conversion of these unit tests here: https://github.com/hashgraph/hedera-services/issues/11560

Reply:
Your next comment is correct. I cloned this test, and the clone uses birth rounds.